### PR TITLE
Completed ds-prep-python-tuples

### PR DESCRIPTION
**Pull request by TroyFaruki**
image

### CODE REVIEW

File: ds-prep-python-tuples/main.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+passenger = (12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print(passenger)
+name = passenger[2]
+print(name)
+age = passenger[-1]
+print(age)
+print(name, age)
```
Comment (TimelikeClosure) [inline ds-prep-python-tuples/main.py]: Please avoid going overboard with the `print` statements. It makes your output more difficult to examine:

---

image

---

image

File: ds-prep-python-tuples/main.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+survived_and_name = passenger[1:3]
+print(survived_and_name)
+gender_and_age = passenger[3:5]
```
Comment (TimelikeClosure) [inline ds-prep-python-tuples/main.py]: For getting `gender_and_age`, please only specify the start of the range:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: ds-prep-python-tuples/main.py (hunk 2 above)
Comment (TroyFaruki) [reply ds-prep-python-tuples/main.py]: Tim from what I understand we are supposed to output gender and age so ('female', 58) is incorrect? I am not sure how to approach this problem unless I utilize a negative indice?

Comment (TroyFaruki) [reply ds-prep-python-tuples/main.py]: image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**